extensions/funqy/funqy-knative-events/deployment/src/test/java/io/quarkus/funqy/test/WithInvalidAttributeFilterTest.java

```diff
@@ -0,0 +1,44 @@
+package io.quarkus.funqy.test;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.spec.JavaArchive;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+import io.quarkus.test.QuarkusUnitTest;
+import io.restassured.RestAssured;
+
+public class WithInvalidAttributeFilterTest {
```

Comment:
I think this name is not descriptive. I would almost say that it can be called overlapping, but that is not true either. The case we are testing here is that two predicates are in sub-set relation. Can you think of any better name, or maybe adding comment would be enough.

Reply:
how about `WithConflictingAttributeFilterTest`?

Reply:
yes, that's better, or maybe `WithAlwaysConflictingAttributeFilterTest`, but that's quite mouthful.

Reply:
renamed to `WithConflictingAttributeFilterTest`